playbooks/templates/tripleo-deploy.sh.j2

```diff
@@ -7,10 +7,10 @@ sudo openstack tripleo deploy \
     --templates \
     --local-ip={{ control_plane_ip }} \
     --control-virtual-ip {{ public_api }} \
-    -e /usr/share/openstack-tripleo-heat-templates/environments/standalone/standalone-tripleo.yaml \
-{% for service in enabled_services %}
+  {% for service in enabled_services %}
     -e {{ service }} \
-{% endfor %}
+  {% endfor %}
+    -e /usr/share/openstack-tripleo-heat-templates/environments/standalone/standalone-tripleo.yaml \
```

Comment:
Needed to move the enabled services up (not sure if absolutely necessary, but it seemed it would not work correctly the other way around. could have been user error.

Reply:
this was not needed in my testing, btw.